jupyterlab/locale/jupyterlab.pot

```diff
@@ -3621,13 +3596,3 @@ msgstr ""
 #: packages/translation-extension/src/index.ts:142
 msgid "Are you sure you want to refresh?"
 msgstr ""
-
-#. obsolete
```

Comment:
I think we need to add support for merging source string to avoid removal of entries like this one? Cc @goanpeca

Reply:
Also, it would be useful if the comment said which was the last version the string was seen so we can make easily remove only the ones that we don't care about anymore later.

If we care about this one which I presume was in 3.0.0 but is not in 3.0.16) I could just edit this PR manualy.

Reply:
Yep for this one a manual update sounds good enough